common/src/main/java/xyz/wagyourtail/jsmacros/client/api/helpers/OptionsHelper.java

```diff
@@ -356,15 +414,19 @@ public OptionsHelper setSmoothCamera(boolean val) {
     /**
      * @since 1.5.0
      * @return 0 for 1st person, 2 for in front.
+     * @deprecated use the respective method in {@link xyz.wagyourtail.jsmacros.client.api.library.impl.FClient#getFullGameOptions()} instead.
      */
+    @Deprecated
     public int getCameraMode() {
         return base.getPerspective().ordinal();
     }
 
     /**
      * @param mode 0: first, 2: front
      * @since 1.5.0
+     * @deprecated use the respective method in {@link xyz.wagyourtail.jsmacros.client.api.library.impl.FClient#getFullGameOptions()} instead.
      */
+    @Deprecated
```

Comment:
I was thinking you'd also just merge the classes...

Reply:
I merged them now.